packages/buckets/src/buckets.spec.ts

```diff
@@ -264,6 +265,13 @@ describe('Buckets...', function () {
       }
       list = await client.listPath(rootKey, '')
       expect(list.item?.items).to.have.length(2) // Includes .textileseed
+
+      try {
+        await client.removePath(rootKey, 'path', {root: wrongRoot})
+        throw wrongError
+      } catch (err) {
+        expect(err.message).to.equal('update is non-fast-forward')
```

Comment:
the proof

Reply:
yeh nice test, good to have this in here.